jupytext/cli.py

```diff
@@ -711,7 +732,11 @@ def write_function(path, fmt):
         log("[jupytext] Sync timestamp of '{}'".format(nb_file))
         os.utime(nb_file, None)
 
-    return 0
+    if args.alert_untracked and is_untracked(nb_dest):
+        log("[jupytext] Output file {} is not tracked in the git index".format(nb_dest))
```

Comment:
This message is going to be seen lots of times. Maybe we should say explicitly that the user needs to call "git add" on that file?

Reply:
Actually, it won't be seen at all unless the pre-commit hook is in verbose mode...